src/MICore/PlatformUtilities.cs

```diff
@@ -131,6 +131,33 @@ public static string GetEnvironmentVariable(this ProcessStartInfo processStartIn
 #endif
             return null;
         }
+
+        public static string UnixPathToWindowsPath(string unixPath)
+        {
+            return unixPath.Replace('/', '\\');
+        }
+
+        public static string WindowsPathToUnixPath(string windowsPath)
+        {
+            return windowsPath.Replace('\\', '/');
+        }
+
+        public static string NormalizeClientPath(string path)
+        {
+            if (string.IsNullOrWhiteSpace(path))
+            {
+                return path;
+            }
+
+            if (IsWindows())
```

Comment:
I think this is the wrong check -- if you are trying to normalize to what we should be returning to the client then you want to reverse this (on Windows we should be calling `UnixPathToWindowsPath`). If you are trying to normalize to what you should be sending to GDB then you want to check the OS of the target rather than the OS of MI Engine.